<launch>

  <node pkg="occupancy_mapping" type="occupancy_mapping" name="occupancy_mapping" output="screen">
      launch-prefix="xterm -e gdb --args"
      <param name="base_frame" value="/bubbleRob" />
      <param name="max_range" value="2" />
      <param name="tolerance" value="0.01" />
      <param name="traverse_threshold" value="0.5" />
      <param name="n_samples" value="1500" />
      <param name="belief_mod" value="1.0" />
      <param name="normal_estimation_radius" value="0.03" />
      <param name="step_function_parameter" value="2.0" />
	 <param name="alpha" value="9.0" />
	 <param name="beta" value="2.0" />
	 <param name="z_threshold" value="0.1" />
    
      <remap from="/occupancy_mapping/scans" to="/vrep/depthSensor"/>
  </node>
    <node respawn="true" pkg="joy"
        type="joy_node" name="joy" >
        
    </node>
</launch>
